markdown_it/_compat.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from typing import Any
 
-if sys.version_info >= (3, 10):
+if sys.version_info >= (3, 10, 0, "release"):
```

Comment:
```suggestion
if sys.version_info >= (3, 10, 0, "final"):
```

It seems like `"release"` is not a valid release level. I think what you want is `"final"`? https://docs.python.org/3/library/sys.html#sys.version_info

Reply:
:clown_face: :facepalm: 

Wow, yes. Great catch @hukkin!

I will cut myself some slack and say that I was fairly sleep deprived when I wrote this.